<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MDIMainWindow</class>
 <widget class="QMainWindow" name="MDIMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MDIMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_close_all"/>
   <addaction name="separator"/>
   <addaction name="action_cut"/>
   <addaction name="action_copy"/>
   <addaction name="action_paste"/>
   <addaction name="action_font"/>
   <addaction name="separator"/>
   <addaction name="action_mdi"/>
   <addaction name="action_cascade"/>
   <addaction name="action_tile"/>
   <addaction name="separator"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_new">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/100.bmp</normaloff>:/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>新建文档</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/122.bmp</normaloff>:/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文档</string>
   </property>
  </action>
  <action name="action_close_all">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/128.bmp</normaloff>:/images/128.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭全部</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/200.bmp</normaloff>:/images/200.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/202.bmp</normaloff>:/images/202.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/204.bmp</normaloff>:/images/204.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_font">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/506.bmp</normaloff>:/images/506.bmp</iconset>
   </property>
   <property name="text">
    <string>字体设置</string>
   </property>
  </action>
  <action name="action_mdi">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/230.bmp</normaloff>:/images/230.bmp</iconset>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
  </action>
  <action name="action_cascade">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/400.bmp</normaloff>:/images/400.bmp</iconset>
   </property>
   <property name="text">
    <string>级联展开</string>
   </property>
  </action>
  <action name="action_tile">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/406.bmp</normaloff>:/images/406.bmp</iconset>
   </property>
   <property name="text">
    <string>平铺展开</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/images/132.bmp</normaloff>:/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MDIMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
